kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonWindowController.java

```diff
@@ -120,6 +126,20 @@ public SkeletonWindowController(EditorController editorController, String docume
         if (editorController.getFxomDocument() != null) {
             editorController.getFxomDocument().sceneGraphRevisionProperty().addListener(fxomDocumentRevisionListener);
         }
+        
+        this.copyAccelerator = new KeyCharacterCombination("c", KeyCombination.SHORTCUT_DOWN);
```

Comment:
Can we replace `KeyCharacterCombination` with `KeyCodeCombination` ?

Reply:
Okay, I'll replace this one. Whats the advantage of `KeyCharacterCombination` over `KeyCodeCombination`?